frontend/src/components/App.jsx

```diff
@@ -40,7 +40,10 @@ class App extends Component {
 }
 
 const mapDispatchToProps = dispatch => ({
-  loadCreatorData: () => dispatch(actions.fetchCreators())
+  loadCreatorData: () => {
```

Comment:
another non-blocking comment. i think i'd prefer to individually call the actions inside componentDidMount, rather than bundle them all together inside the `loadCreatorData` dispatch. the latter gives the impression that they're only getting creator data, but actually they're also getting collection data.